pom.xml

```diff
@@ -151,6 +151,9 @@
     <module>parquet-scala</module>
     <module>parquet-thrift</module>
     <module>parquet-hadoop-bundle</module>
+    <!--
+    <module>plugins/parquet-encoding-vector</module>
```

Comment:
This means user has to remove the comment to enable it which seems not very friendly. We can use `<profile>` to enable module optionally.

Reply:
I have removed it, user need go in plugins/parquet-encoding-vector and build with maven